extensions/wikia/CreateNewWiki/classes/Starters.class.php

```diff
@@ -35,15 +39,22 @@ class Starters {
 	];
 
 	/**
-	 * Get the starter database for a given language code
+	 * Get the starter database for a given language code and vertical
 	 *
 	 * @param string $lang language code
+	 * @param int $vertical vertical id
+	 *
 	 * @return string
 	 */
-	public static function getStarterByLanguage( $lang ) {
-		return ( isset( self::$mStarters[ $lang ] ) )
-			? self::$mStarters[ $lang ]
-			: self::$mStarters[ '*' ];
+	public static function getStarterByLanguageAndVertical( $lang, int $vertical ) {
+		// TODO Clean up in XW-2393
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)